Makefile

```diff
@@ -116,6 +116,12 @@ run: build ## Run
 build-image:
 	DOCKER_BUILDKIT=1 $(OCI_RUNTIME) build -t $(DOCKER_IMG):$(DOCKER_TAG) -f contrib/docker/Dockerfile .
 
+.PHONY: build-main-and-sha
+build-main-and-sha: build-image
+	echo "FROM $(DOCKER_IMG):$(DOCKER_TAG)" > tmp.Dockerfile && \
```

Comment:
@jotak I am not sure that I get the process here .... don't we need to execute `build-image` so we will have the `FROM $(DOCKER_IMG):$(DOCKER_TAG)` available as a prereq for `build-main-and-sha` ??? without running that why should we have that image and we can use that as "FROM" ?

Reply:
but that's what it does: `build-main-and-sha: build-image`
Am I missing something?

Reply:
@jotak No, it was me that did not understand ... now I get it + the new version is much much more simple and easy to understand ... :-) ...I like think that this is very simple and good now :-)